### DISCUSSION TITLE
Support for PostgreSQL database v16.x

### DISCUSSION THREAD

**Discussion by a-z-i-z** [apache/superset#27546] [Ideas]
As per the [documentation](https://superset.apache.org/docs/installation/configuring-superset/#using-a-production-metastore), it looks like superset doesn't supports PostgreSQL v16.x. Can we please have a support for this version ?

**Comment by raphaelauv** [comment]
I use superset with postgres 16.2 , it works well

**Reply by a-z-i-z** [reply]
@raphaelauv So it means superset docs aren't updated yet.

**Reply by gansoper** [reply]
Do you mean DB that contains your data source or one that comes with superset? As I can see superset installation pulls 15 version of postgres and using 16, or even latest provides installation error

**Reply by raphaelauv** [reply]
I'm talking about superset database ( not a data source )

**Comment by RealGreenDragon** [comment]
Opened PR #32597 to add PostgreSQL v16 as supported superset database.
